src/status_im/bots/subs.cljs

```diff
@@ -2,13 +2,11 @@
   (:require [re-frame.core :as re-frame]
             [status-im.chat.models.input :as input-model]))
 
-(re-frame/reg-sub
-  :bot-db
+(re-frame/reg-sub :bot-db
```

Comment:
in cursive this will be aligned bad, fn will be under :bot-db, let's use previous alignment

Reply:
I will revert it